README.md

```diff
@@ -4,3 +4,96 @@ Codespaces devcontainer templates
+
+This is a feature complete version of Wallaroo Community with the following exceptions:
+
+* Connections from the Wallaroo SDK to the Wallaroo Community instance are only available from the Wallaroo Community instance's JupyterHub service, launched from the Wallaroo Dashboard.
+
+## Prerequisites
+
+* **GitHub Account**:  [A GitHub account](https://github.com/).
+* **GitHub Codespaces**:  See the [GitHub Codespaces Cost Calculator](https://github.com/pricing/calculator) for details on services costs.
+* **Wallaroo Community License**:  Available for free from the [Wallaroo Community Portal page](https://portal.wallaroo.community/).
+
+## Installation Steps
+
+1. From the [Wallaroo Community Portal page](https://portal.wallaroo.community/), follow the registration instructions.  If a Wallaroo Community license already exists for this account, enter the same name and email address to retrieve the registration link.
+1. Download the Wallaroo Community license.  This provides both the Wallaroo Community license in YAML format and the base64 encoded version.  Store both in a secure location.  The base64 ended version will be used in this process.
+1. From the License Download page, select **GitHub Codespaces**.  Follow the link to the Wallaroo Community Codespaces installation template.
+2. Set the following:
+
+    ![Wallaroo GitHub Install Options](./images/wallaroo-community-github-codespace/wallaroo-community-github-codespace-configuration.png)
+
+    1. **Dev container configuration**:  Set to **Wallaroo Codespaces Data Science**.
+    2. **Region**:  Select the region where the codespace will run.  This should be the region geographically closest to the end user.
+        1. Europe West
+        2. Southeast Asia
+        3. US West
+        4. US East
+    3. **Machine Type**:  Set the number of CPUs to use.  Leave as 16-cores - anything less the containers will not run.
+    4. **Wallaroo Community License**:  Provide the base64 version of the Wallaroo license here.
+
+3. When complete, select **Create Codespace**.  Wait until the process is complete - this usually takes about 15 minutes.
+
+To see the installation process in action, use one of the following options:
+
+* Enter either `Control-Shift-P` or `Command-Shift-P` (for MacOS systems) and select **Codespaces: View Creation Log**.
+* Select from the lower left corner **Codespaces** and then **View Creation Log**.
+
+![Test](./images/wallaroo-community-github-codespace/github-spaces-view-creation-log.gif)
+
+Once the entry "Finished configuring codespace" is displayed, the Wallaroo Community installation is complete.
+
+### Network Configure
+
+Before connecting to the Wallaroo Community instance, the following codespaces ports must be set to **public**.  This step is not required for users who will be the only one accessing Wallaroo Community though their codespace.
```

Comment:
This a reminder that we'll want to change this to required for everyone

Reply:
With the new updates, I'll just take that out.